docs/conf/portland/2018/press-kit.rst

```diff
@@ -0,0 +1,81 @@
+
+  I'd like to attend Write the Docs in Portland, 6-8 May 2018. A three-day event that explores the art and science of documentation. The conference covers topics related to documentation in the software industry.
+
+  Write the Docs brings everyone who writes the docs together in the same room: Writers, Developers, Developer Relations, and Support Staff, which makes it an ideal networking opportunity .
+
+  There is a single talk track, a parallel unconference event and a writing day. The `sessions from 2017 <http://www.writethedocs.org/conf/na/2017/speakers/>`_ provide a reasonable idea of what to expect.
+
+  Cost:
+
+  * ticket $450
+  * travel
+  * accomodation
+
+  Benefits:
+
+  * networking with other documentarians
+  * exposure to new or state of the art ideas/techniques/products
+  * a chance to contribute to discussions and working groups
+  * find solutions to problems you're facing at work
+
+  Sincerely,
+  [your name]
+
+Good Things People Have Said About Us
+-------------------------------------
+
+.. raw:: html
+
+  <blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">I&#39;m finding myself really sad that today isn&#39;t a <a href="https://twitter.com/hashtag/writethedocs?src=hash&amp;ref_src=twsrc%5Etfw">#writethedocs</a> day today too. I mean yes, today I can literally write the docs, but still.</p>&mdash; Diana Potter (@drpotter) <a href="https://twitter.com/drpotter/status/601133512205291520?ref_src=twsrc%5Etfw">May 20, 2015</a></blockquote>
+  <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+  <blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/hashtag/writethedocs?src=hash&amp;ref_src=twsrc%5Etfw">#writethedocs</a> ended yesterday. It was such a great conference! <a href="https://t.co/uhyEIYrbTV">https://t.co/uhyEIYrbTV</a></p>&mdash; 🌟 Aleen vs. the Forces of Evil 🌟 (@Aleen) <a href="https://twitter.com/Aleen/status/601111911791534081?ref_src=twsrc%5Etfw">May 20, 2015</a></blockquote>
+  <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+  <blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">On my way home from another great <a href="https://twitter.com/hashtag/writethedocs?src=hash&amp;ref_src=twsrc%5Etfw">#writethedocs</a>. Already working on ways to apply what I learned. If you care about docs, be here next time!</p>&mdash; Daniel D. Beck (@ddbeck) <a href="https://twitter.com/ddbeck/status/601042665744957440?ref_src=twsrc%5Etfw">May 20, 2015</a></blockquote>
+  <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Useful Numbers
+---------------
+
+We have some stats about attendees, membership, and website hits over on the following blog posts:
+
+- :doc:`/blog/write-the-docs-2017-stats/`
+- :doc:`/blog/write-the-docs-2016-year-in-review/`
+
+
+Images and Logos
+-------------------
```

Comment:
I'd love to put in some group photos and other useful "use this to talk about us" photos. A lot of people post crappy photos in articles, so it would be cool to have a set we recommend.

Reply:
I've asked @goatlady to pick us out 3 of her faves, including a group shot and a venue shot.

Reply:
What do we think of these: 

1: https://www.flickr.com/photos/writethedocs/34495135662/in/album-72157683817839465/
2: https://www.flickr.com/photos/writethedocs/33849416753/in/album-72157683817839465/
3: https://www.flickr.com/photos/writethedocs/33856825394/in/album-72157683817839465/

Reply:
👍